src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -1147,7 +1138,7 @@ StDebugger >> stack [
 { #category : 'specs' }
 StDebugger >> stackAndCodeLayout [
 
-	^ SpPanedLayout newTopToBottom
+	^ SpPanedLayout newLeftToRight
 		positionOfSlider: 30 percent;
 		add: self stackLayout;
 		add: self codeLayout;
```

Comment:
This is actually the only change right?
All the other diffs are just code reformating? (hard to see)